### PR TITLE
bpo-34762: Fix contextvars C API to use PyObject* pointer types.

### PR DESCRIPTION
**Pull request by 1st1**
* [x] **Note:** if this PR is accepted I'll update the docs before merging it.

https://bugs.python.org/issue34762

### CODE REVIEW

**Review by vstinner — APPROVED** [review]
LGTM.

**Review by rhettinger — APPROVED** [review]

**Review by asvetlov — APPROVED** [review]

**Comment by 1st1** [comment]
This is how the docs section looks when rendered:

screen shot 2018-09-21 at 2 54 11 pm

File: Doc/c-api/contextvars.rst [hunk 1]
```diff
@@ -5,6 +5,25 @@
 Context Variables Objects
 -------------------------
 
+.. _contextvarsobjects_pointertype_change:
+.. versionchanged:: 3.7.1
+
+   .. note::
+
+      In Python 3.7.1 signatures of all context variables
```
Comment (elprans) [inline Doc/c-api/contextvars.rst]: *the* signatures

**Review by elprans — APPROVED** [review]

**Comment by 1st1** [comment]
Thanks everybody for reviews!

**Pull request merged**